src/qt/guiutil.cpp

```diff
@@ -423,7 +423,7 @@ bool ToolTipToRichTextFilter::eventFilter(QObject *obj, QEvent *evt)
         QString tooltip = widget->toolTip();
         if(tooltip.size() > size_threshold && !tooltip.startsWith("<qt") && !Qt::mightBeRichText(tooltip))
         {
-            // Envelop with <qt></qt> to make sure Qt detects this as rich text
+            // Envelope with <qt></qt> to make sure Qt detects this as rich text
```

Comment:
Nit. I think this should be envelop rather than envelope. Envelope is a noun, envelop is a verb.

Reply:
Done, thanks! That seemed misspelled to me but you are correct.